src/content/docs/release-notes/agent-release-notes/ruby-release-notes/ruby-agent-800.mdx

```diff
@@ -3,6 +3,9 @@ subject: Ruby agent
 releaseDate: '2021-09-22'
 version: 8.0.0
 downloadLink: 'https://rubygems.org/downloads/newrelic_rpm-8.0.0.gem'
+features: ["Distributed tracing enabled by default", "Cross Application is deprecated and disabled by default", "Update configuration default value for `span_events.max_samples_stored` from 1000 to 2000", "'add_method_tracer' now uses prepend chaining", "Add support for ARM64/Graviton2 platform using Ruby 3.0.2+"]
+bugs: ["Correctly load configuration options from newrelic.yml"]
```

Comment:
Were there certain config options fixed? This one feels a bit vague/concerning

Reply:
[Specified](https://github.com/newrelic/docs-website/pull/11703/commits/f755804b3a1fea8abcd87d362cc92a4c34c3ea03#diff-907e1504529aaf59a55f966279bab8b4353b1cafc882eca69c4859c182f13a4fR7) the config options that were fixed